packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
+                  : minValue}
+              </Typography>
+            </Collapse>
+          </Box>
+        </TableCell>
+        <CenteredTableCell sx={{ textAlign: "center" }}>
+          <Checkbox
+            checked={required}
+            onChange={(event) => {
+              update(propertyIndex, {
+                ...draftProperty,
+                required: event.target.checked,
+              });
+            }}
+          />
+        </CenteredTableCell>
+        <CenteredTableCell sx={{ px: "0px !important" }}>
+          <TextField
+            placeholder="Add default value"
+            sx={{
+              width: "100%",
+              [`.${tableRowClasses.root}:not(:hover) & .${outlinedInputClasses.root}:not(:focus-within)`]:
+                {
+                  boxShadow: "none",
+                  [`.${outlinedInputClasses.notchedOutline}`]: {
+                    borderColor: "transparent",
+                  },
+                  [`.${outlinedInputClasses.input}::placeholder`]: {
+                    color: "transparent",
+                  },
                 },
-              },
-          }}
-          inputProps={{ sx: { textOverflow: "ellipsis" } }}
-        />
-      </TableCell>
-      <TableCell>
-        <PropertyMenu onRemove={onRemove} property={property} />
-      </TableCell>
-    </TableRow>
-  );
+            }}
+            inputProps={{ sx: { textOverflow: "ellipsis" } }}
+          />
+        </CenteredTableCell>
+        <TableCell>
+          <PropertyMenu onRemove={onRemove} property={property} />
+        </TableCell>
+      </TableRow>
+      <Popper
+        open={multipleValuesMenuOpen}
+        anchorEl={anchorEl}
+        container={anchorEl}
+        placement="bottom"
+        sx={{ width: 1, zIndex: 1 }}
+        transition
+      >
+        {({ TransitionProps }) => {
+          return (
+            <ClickAwayListener
+              onClickAway={() => setMultipleValuesMenuOpen(false)}
+            >
+              <Fade {...TransitionProps}>
+                <Box
+                  sx={({ palette }) => ({
+                    border: 1,
+                    p: 1.5,
+                    background: palette.white,
+                    borderColor: palette.gray[30],
+                  })}
+                >
+                  <TextField
+                    type="number"
+                    size="small"
+                    label="Minimum"
+                    value={minValue ?? 0}
+                    onChange={(event) => {
+                      const value = parseInt(event.target.value, 10);
+                      if (value >= 0) {
+                        update(propertyIndex, {
+                          ...draftProperty,
+                          minValue: value,
+                          maxValue:
+                            !maxValue || value > maxValue ? value : maxValue,
+                        });
+                      }
+                    }}
```

Comment:
This should also use `register`. You can subscribe to `minValue` with `useWatch` in order to enforce the `maxValue` constraint here.

Reply:
Done in 669f2a5700d2e0a6385302b661eb077a24186144. I enforced the `maxValue` constraint through `useEffect` but had to keep `minValue` out of the deps array. Let me know if there's a better way to do this, please.